platform/macos/src/MGLMapView.mm

```diff
@@ -1633,13 +1633,14 @@ - (BOOL)gestureRecognizer:(NSGestureRecognizer *)gestureRecognizer shouldAttempt
 #pragma mark Keyboard events
 
 - (void)keyDown:(NSEvent *)event {
-    if (event.modifierFlags & NSNumericPadKeyMask) {
-        // This is the recommended way to handle arrow key presses, causing
-        // methods like -moveUp: and -moveToBeginningOfParagraph: to be called
-        // for various standard keybindings.
-        [self interpretKeyEvents:@[event]];
-    } else {
-        [super keyDown:event];
```

Comment:
`-keyDown:` and `-insertText:` should call through to the superclass’s implementation for any unhandled keys/text so that, for example, pressing <kbd>M</kbd> beeps and pressing ⇥ or ⇧⇥ cycles to another responder.